ui/src/components/Inventory/InventoryCreateModal.js

```diff
@@ -0,0 +1,127 @@
+import Button from '@material-ui/core/Button'
```

Comment:
This is looking good so far. The only thing I'm seeing that's missing is the validation to indicate which fields are required